### PR TITLE
Comments

### PR DESCRIPTION
**Pull request by sergiomonz**
Creación de modal para comentarios de lugar y de evento.

### CODE REVIEW

File: dashboard/src/components/Comment.vue [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    showModal: false,
+    postModel: {
+      comment: "",
+      score: 5.0,
+    },
+    parametros:""
+  }),
+  methods: {
+    register() {
+      console.log(this.composite_key);
```
Comment (1701722767) [inline dashboard/src/components/Comment.vue]: delete this

File: dashboard/src/components/Comment.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        })
+        .catch((err) => {
+          this.loading = false;
+          notification({
+            message: "Ocurrió un error al hacer la petición",
+          });
+        });
+    },
+
+    getKeysEvent(event_id, category_id){
```
Comment (1701722767) [inline dashboard/src/components/Comment.vue]: delete this

**Review by 1701722767 — CHANGES_REQUESTED** [review]

**Review by nicolascr181 — APPROVED** [review]

File: dashboard/src/components/Comment.vue (hunk 1 above)
Comment (sergiomonz) [reply dashboard/src/components/Comment.vue]: Done

File: dashboard/src/components/Comment.vue (hunk 2 above)
Comment (sergiomonz) [reply dashboard/src/components/Comment.vue]: Done

File: dashboard/src/helpers/Request.js [hunk 3]
```diff
@@ -2,10 +2,15 @@ import { getToken } from "@/services/Auth.js";
 import { API_URL } from "../config/Constants.js";
 import { TOKEN_KEY } from "../config/Constants.js";
 
-export const postJSON = async function (path = "", data = {}, needToken) {
-  let url = API_URL + path;
+export const postJSON = async function (path = "", data = {}, needToken, params=null) {
+  let url = APP_HOST + path;
```
Comment (1701722767) [inline dashboard/src/helpers/Request.js]: change this to API_URL

**Review by 1701722767 — CHANGES_REQUESTED** [review]

File: dashboard/src/helpers/Request.js (hunk 3 above)
Comment (sergiomonz) [reply dashboard/src/helpers/Request.js]: Done

**Review by 1701722767 — APPROVED** [review]

**Pull request merged**